src/main/java/com/epam/internship/carrental/car/CarUtilities.java

```diff
@@ -0,0 +1,43 @@
+package com.epam.internship.carrental.car;
+
+/**
+ * Utility class for the car class.
+ */
+public final class CarUtilities {
+
+    /**
+     * Final constructor to prevent instantiation.
+     */
+    private CarUtilities() {
+    }
+
+    /**
+     * Updates the car defined as the first parameter, with the parameters of the second car
+     * which is the second parameter.
+     *
+     * @param carToUpdate  updateable car
+     * @param newCarParams car with the new parameters
+     * @return updated Car object
+     */
+    public static Car updateCar(final Car carToUpdate, final Car newCarParams) {
+        if (newCarParams.getCarType() != null) {
```

Comment:
Use defensive style. Check if the actual parameters are not `null` to avoid NPE.